### PR TITLE
Acl anthology formatting

### PR DESCRIPTION
**Pull request by ToluClassics**
Script for formatting bib collection file before indexing

### CODE REVIEW

File: scripts/aclanthology/cleanup_bibcollection.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                category[-1] = '"' + category[-1] + '"'
+            category = " = ".join(category)
+        all_cat.append(category)
+
+    return ",\n".join(all_cat)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Bibtex Parser")
+    parser.add_argument(
```
Comment (lintool) [inline scripts/aclanthology/cleanup_bibcollection.py:31]: fit on a single line?

File: scripts/aclanthology/cleanup_bibcollection.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import os
```
Comment (lintool) [inline scripts/aclanthology/cleanup_bibcollection.py:1]: organize imports?

**Review by lintool — APPROVED** [review]
minor comments, don't need to review it again...

File: scripts/aclanthology/cleanup_bibcollection.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    with open(args.input_path) as f:
+        text = []
+        for line in f:
+            if not line.isspace():
+                text.append(line)
+        text = "".join(text)
+        text = text.split("\n}\n")
+        text = [bib + "\n}" for bib in text[:-1]]
+
+    bib_collection = []
```
Comment (AlexWang000) [inline scripts/aclanthology/cleanup_bibcollection.py:48]: Please use this, the current script has a bug
```
bib_collection = []
    for index, item in enumerate(tqdm(text, desc="parsing bib collection")):
        try:
            parsed = bibtexparser.loads(format(item))
            bib_collection.append(parsed.entries[0])
        except:
            print(f"Unable to Parse document {index}:\n {item}\n")
```

**Pull request closed without merging**